### PR TITLE
Synchronization to S3

### PR DESCRIPTION
**Pull request by xtruan**
Added support to synchronize database dumps to S3 using `s3cmd` tool.

### CODE REVIEW

**Review by prophund — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,7 +24,13 @@ runs every day at 1:00 am).
 | `PREFIX`                 | `None`       | Optional   | dump                  | Optionally, prefix for dump files                                             |
 | `PGDUMP`                 | `None`       | Optional   | /dump                 | Optionally, define a different location to dump your backups.                 |
 | `COMMAND`                | `None`       | Optional   | `dump-cron` | Options: `dump` dumps the database and exit, `dump-cron` creates a cron job and runs    |
-| `POSTGRES_PASSWORD_FILE` | `None`       | Recomended | `None`                | Location of the password file. Overrides `POSTGRES_PASSWORD` and `PGPASSWORD` |
+| `POSTGRES_PASSWORD_FILE` | `None`       | Recommended | `None`               | Location of the password file. Overrides `POSTGRES_PASSWORD` and `PGPASSWORD` |
+| `S3_ACCESS_KEY`          | `None`       | Required for S3 sync   | `None` | Access key to connect to S3. Example: `AKIAIOSFODNN7EXAMPLE`                     |
```
Comment (Martlark) [inline README.md:28]: Excellent work.  I was considering this and you've saved me the trouble.  s3cmd is so easy

File: README.md [hunk 2]
```diff
@@ -68,7 +76,7 @@ services:
       - ./persistent/data:/dump
 ```
 
-### Example asswords file
+### Example passwords file
```
Comment (Martlark) [inline README.md:79]: Thanks

File: entrypoint.sh [hunk 3]
```diff
@@ -52,13 +52,29 @@ elif [[ "${COMMAND}" == 'dump-cron' ]]; then
...
+
+    if [[ ! -z "${S3_SECRET_KEY}" ]]; then
+        CRON_ENV="$CRON_ENV\nS3_SECRET_KEY='${S3_SECRET_KEY}'"
+    fi
+
+    if [[ ! -z "${S3_BUCKET_PATH}" ]]; then
+        CRON_ENV="$CRON_ENV\nS3_BUCKET_PATH='${S3_BUCKET_PATH}'"
+    fi
+
+    echo -e "$CRON_ENV\n$CRON_SCHEDULE" "/dump.sh >> ${PGDUMP}/dump-log" | crontab -
```
Comment (Martlark) [inline entrypoint.sh:74]: Nicely fixed. My previous thing was a quicky.

**Review by Martlark — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xtruan) [reply README.md:28]: Hey, thank you! This is an extremely useful little tool. I'm using it to backup metadata stored in Postgres for some big datasets I manage.

**Pull request merged**